docs/Documentation/Systems/Kestrel/Running/example_sbatch.md

```diff
@@ -0,0 +1,174 @@
+
+    ```
+    #!/bin/bash
+    #SBATCH --nodes=2             # Use 2 nodes
+    #SBATCH --partition=gpu-h100
+    #SBATCH --time=00:20:00       # Set a 20 minute time limit
+    #SBATCH --ntasks-per-node=2   # Maximum CPU cores for job 
+    #SBATCH --gres=gpu:4          # GPU request 
+
+    export CUDA_VISIBLE_DEVICES=0,1,2,3
+
+    # Enable access to new modules for running on GPUs
+    source /nopt/nrel/apps/gpu_stack/env_cpe23.sh
+
+    # Load modules
+    module purge
+    ml craype-x86-genoa  # Module to set optimizations for CPUs on GPU nodes
+
+    # Run program
+    cd /scratch/$USER 
+    srun my_graphics_intensive_scripting 
+    ```
+    GPU nodes can be shared so you may request fewer than all 4 GPUs on a node. When doing so, you must also request appropriate CPU cores and memory with the `--ntasks-per-node=<NUM_CPUS>` and `--mem=<MEMORY_REQUEST>` flags, respectively.
+    
+    ```
+    #!/bin/bash
+    #SBATCH --nodes=2             # Use 2 nodes
+    #SBATCH --partition=gpu-h100
+    #SBATCH --time=00:20:00       # Set a 20 minute time limit
+    #SBATCH --ntasks-per-node=2   # Maximum CPU cores for job 
+    #SBATCH --gres=gpu:2          # GPU request 
+    #SBATCH --mem=184000          # Standard partition (192GB nodes) 
+
+    export CUDA_VISIBLE_DEVICES=0,1
+
+    # Enable access to new modules for running on GPUs
+    source /nopt/nrel/apps/gpu_stack/env_cpe23.sh
+
+    # Load modules
+    module purge
+    ml craype-x86-genoa  # Module to set optimizations for CPUs on GPU nodes
+
+    # Run program
+    cd /scratch/$USER 
+    srun my_graphics_intensive_scripting 
+    ```
+    *Currently, `source /nopt/nrel/apps/gpu_stack/env_cpe23.sh` is necessary to access GPU modules. This is subject to change as the system improves.*
+
+    The `CUDA_VISIBLE_DEVICES` environment variable specifies the GPU(s) your programs will run on, and is included for the sake of completeness. Read more about this environment variable [here](https://developer.nvidia.com/blog/cuda-pro-tip-control-gpu-visibility-cuda_visible_devices/).
+
+??? example "Sample batch script to utilize Local Disk ($TMPDIR)"
+    ```
+    #!/bin/bash 
+    #SBATCH --ntasks=36                # CPU cores requested for job 
+    #SBATCH --nodes=1                  # Keeep all cores on the same node 
+    #SBATCH --time=01-00               # Job should run for up to 1 day (for example) 
+    #SBATCH --tmp=20TB                 # Request minimum 20TB local disk 
+
+    # Copy files into $TMPDIR 
+    cp /scratch/<userid>/myfiles* $TMPDIR 
+
+    srun ./my_parallel_readwrite_program -input-options $TMPDIR/myfiles  # use your application's commands  
+    ```
+    *`$TMPDIR` is a preset variable that points to `/tmp/scratch/<JOB_ID>`. Be sure to use the flag `SBATCH --tmp=<LOCAL_DISK_REQUEST>` or your job will use RAM.*
+
+??? example "Sample batch script for an MPI job (CPU and GPU)"
+    The default module for running MPI jobs, PrgENV-cray is automatically loaded for all users.
```

Comment:
I think it would be best to put CPU and GPU MPI examples in separate boxes, GPU MPI can be left out for now since we haven't officially made GPU nodes available